tests/helpers/test_device_registry.py

```diff
@@ -1336,6 +1344,7 @@ async def test_update(
         "device_id": entry.id,
         "changes": {
             "area_id": None,
+            "connections": {("mac", "12:34:56:ab:cd:ef")},
```

Comment:
Would be good to have multiple connections and than verify it can be replaced with no connections or a single connection